### PR TITLE
Bump ghproxy, branchprotector, commenter, and label_sync with Prow.

### PR DESCRIPTION
**Pull request by cjwagner**
Steps: 
- [x] Switch `branchprotector` and `label_sync` from k8s cronjobs to Prow periodic jobs. (These are only cronjobs for legacy reasons AFAIK.) https://github.com/kubernetes/test-infra/pull/11841
- [x] Make Prow bump the image tags for `branchprotector`, `label_sync`, and `commenter` ProwJobs in the `config/` dir.

/assign @krzyzacy @stevekuznetsov

### CODE REVIEW

**Comment by cjwagner** [comment]
> (assume you tried it locally)

I did, but I'm thinking now it might be better to switch `branchprotector` and `label_sync` to use periodic ProwJobs first since those will be bumped differently anyways. Let me get that PRed first.

/lgtm cancel

**Comment by cjwagner** [comment]
This is ready for review now. Image changes are in separate commits in case we need to revert.

File: config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ periodics:
   decorate: true
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
```
Comment (stevekuznetsov) [inline config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml]: Commands on these need to be updated

Comment (cjwagner) [reply config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml]: Fixed.

**Comment by stevekuznetsov** [comment]
/lgtm
Gonna let on-call cancel hold

**Comment by Katharine** [comment]
sure, why not.

/hold cancel

**Pull request merged**